crates/interledger-stream/src/packet.rs

```diff
@@ -875,7 +942,17 @@ impl<'a> SerializableFrame<'a> for StreamDataBlockedFrame {
 fn saturating_read_var_uint<'a>(reader: &mut impl BufOerExt<'a>) -> Result<u64, ParseError> {
     if reader.peek_var_octet_string()?.len() > 8 {
         reader.skip_var_octet_string()?;
-        Ok(u64::MAX)
+
+        if cfg!(feature = "roundtrip-only") {
+            // This is needed because the returned value u64::MAX
+            // will make rounttrip fail, i.e. ByteMut::from(packet)
```

Comment:
```suggestion
            // will make roundtrip fail, i.e. BytesMut::from(packet)
```
(deleted previous version of this comment just in case it shows up strange in the history)